portiaui/app/templates/components/start-url-options.hbs

```diff
@@ -0,0 +1,42 @@
+{{#list-item-text class="title fragments-title"}}
+    Fragments
+    {{#help-icon}}
+        Fragments
+        <p>
```

Comment:
The wording is a little bit technical maybe:

    A generated url is made up of one or more fragments. A fragment can be a single value, a list of different values or a range of numbers.

    Using fragments you can create a large number of urls.